### PR TITLE
Update free-download-manager from 5.1.38 to 6.9.1

### PR DESCRIPTION
**Pull request by suschizu**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/free-download-manager.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask 'free-download-manager' do
-  version '5.1.38'
+  version '6.9.1'
   sha256 :no_check # required as upstream package is updated in-place
 
   url "https://dn3.freedownloadmanager.org/#{version.major}/#{version.major_minor}-latest/fdm.dmg"
```
Comment (adidalal) [inline Casks/free-download-manager.rb:5]: `url "https://dn3.freedownloadmanager.org/#{version.major}/latest/fdm.dmg"`

From
```sh
curl -q 'https://up.freedownloadmanager.org/fdm6/checkupdate/?os=osx&osversion=10.15&osarchitecture=x86_64&architecture=x86_64&version=6.9.0.2947&locale=en_US&initiator=2&initiatorId=0' | jq '.downloadURL'
```

**Comment by li1191863273** [comment]
xxx@MacBookPro   ~  brew cask install free-download-manager                  SIGINT(2) ↵  1677  20:05:16 
Updating Homebrew...
==> Downloading https://dn3.freedownloadmanager.org/6/latest/fdm.dmg
##O#- #
curl: (60) SSL certificate problem: certificate has expired
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
Error: Download failed on Cask 'free-download-manager' with message: Download failed: https://dn3.freedownloadmanager.org/6/latest/fdm.dmg

**Comment by core-code** [comment]
thats weird because neither Safari nor Chrome complain about the certificate. any ideas @vitorgalvao ?

**Comment by vitorgalvao** [comment]
Related to https://github.com/Homebrew/homebrew-cask/issues/83481.

**Comment by ran-dall** [comment]
@core-code You can run a PR check locally to verify and force using `brew`'s `cURL` to get past this problem. I gave some instructions to @suschizu a few days ago on how you can do that. Just an FYI.

**Comment by core-code** [comment]
thanks there. i'll try it on 'miro' which is also giving problems today :)

**Comment by li1191863273** [comment]
URL_SSL_BACKEND=secure-transport HOMEBREW_NO_ENV_FILTERING=1 brew cask install free-download-manager

**Pull request merged**